### PR TITLE
Feature/grow page

### PR DESCRIPTION
**Pull request by s6thgehr**
Enable transferring gSOL to another address.

### CODE REVIEW

File: packages/app/src/grow/components/PartnerApp.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import clx from "classnames";
+import { type FC, type ReactNode } from "react";
+
+interface Props {
+  children?: ReactNode;
+  className?: string;
+}
+
+const PartnerApp: FC<Props> = ({ children, className }) => (
```
Comment (ronnyhaase) [inline packages/app/src/grow/components/PartnerApp.tsx:9]: I think we have this  info box component elsewhere, too, so maybe give it a more generic name and move it to common?

File: packages/app/src/common/components/AmountInput.tsx [hunk 2]
```diff
@@ -18,6 +19,7 @@ interface AmountInputProps {
   setAmount: (amountStr: string) => void;
   setValid: (valid: boolean) => void;
   mode: UIMode;
```
Comment (ronnyhaase) [inline packages/app/src/common/components/AmountInput.tsx:21]: Didn't we want to get rid of it? Maybe we should actually rename the values to SEND and RECEIVE, where 0.1 is deducted on SEND

**Review by ronnyhaase — APPROVED** [review]
Just two small comments we can also tackle later

File: packages/app/src/common/components/AmountInput.tsx (hunk 2 above)
Comment (s6thgehr) [reply packages/app/src/common/components/AmountInput.tsx:21]: Yeah, but I didn't know how.

**Pull request merged**